site/content/en/blog/releases/1.2.0-rc.md

```diff
@@ -0,0 +1,59 @@
+- Update the default gRPC and HTTP ports for the sdkserver for the 1.2.0 release [\#1180](https://github.com/googleforgames/agones/pull/1180) ([roberthbailey](https://github.com/roberthbailey))
+- Update terraform configs to use Kubernetes 1.13 [\#1179](https://github.com/googleforgames/agones/pull/1179) ([roberthbailey](https://github.com/roberthbailey))
+- Update documentation for the 1.2 release to use Kubernetes 1.13. [\#1178](https://github.com/googleforgames/agones/pull/1178) ([roberthbailey](https://github.com/roberthbailey))
+- Update minikube and KIND developer tooling to Kubernetes 1.13 [\#1177](https://github.com/googleforgames/agones/pull/1177) ([roberthbailey](https://github.com/roberthbailey))
+- Update the prow cluster to use Kubernetes 1.13 [\#1176](https://github.com/googleforgames/agones/pull/1176) ([roberthbailey](https://github.com/roberthbailey))
+- Update GKE development tooling to Kubernetes 1.13 [\#1175](https://github.com/googleforgames/agones/pull/1175) ([roberthbailey](https://github.com/roberthbailey))
+
+**Implemented enhancements:**
+
+- Upgrade to Kubernetes 1.13 [\#1044](https://github.com/googleforgames/agones/issues/1044)
+- Local SDK Server should update internal GameServer on Ready\(\), Allocate\(\) etc. [\#958](https://github.com/googleforgames/agones/issues/958)
+- Error checking of attributes on feature shortcode [\#1205](https://github.com/googleforgames/agones/pull/1205) ([markmandel](https://github.com/markmandel))
+- Implement Reserve for Unity [\#1193](https://github.com/googleforgames/agones/pull/1193) ([markmandel](https://github.com/markmandel))
+- Add ImagePullSecrets for Allocator and Ping \(helm\) [\#1190](https://github.com/googleforgames/agones/pull/1190) ([aLekSer](https://github.com/aLekSer))
+- Implementation of Unity SDK.Connect\(\) [\#1181](https://github.com/googleforgames/agones/pull/1181) ([markmandel](https://github.com/markmandel))
+- Implementation of GameServer\(\) for Unity [\#1169](https://github.com/googleforgames/agones/pull/1169) ([markmandel](https://github.com/markmandel))
+- Add Spec and Health data to default local sdk GameServer [\#1166](https://github.com/googleforgames/agones/pull/1166) ([markmandel](https://github.com/markmandel))
+- Refresh client CA certificate if changed [\#1145](https://github.com/googleforgames/agones/pull/1145) ([pooneh-m](https://github.com/pooneh-m))
+
+Documentation: https://development.agones.dev/site/
+
+See <a href="https://github.com/googleforgames/agones/blob/release-1.2.0-rc/CHANGELOG.md" data-proofer-ignore>CHANGELOG
+</a> for more details on changes.
+
+Images available with this release:
+
+- [gcr.io/agones-images/agones-controller:1.2.0-rc](https://gcr.io/agones-images/agones-controller:1.2.0-rc)
+- [gcr.io/agones-images/agones-sdk:1.2.0-rc](https://gcr.io/agones-images/agones-sdk:1.2.0-rc)
+- [gcr.io/agones-images/agones-ping:1.2.0-rc](https://gcr.io/agones-images/agones-ping:1.2.0-rc)
```

Comment:
how about agones-allocator?

Reply:
Welp! Yep, that should be there. Adding.